app/controllers/hosts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class HostsController < ActionController::API
   def index
     @hosts = Host.with_cname_or_ip_address.order(:hostname)
-
+    expires_now
```

Comment:
Is there a risk someone could DOS the `https://transition.publishing.service.gov.uk/hosts.json` endpoint? Would it be safer to set a really tiny cache instead, e.g. 1 second?